test/src/test/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuildersFormLogoutTests.java

```diff
@@ -71,4 +80,34 @@ public void customWithUriVars() {
 		assertThat(request.getRequestURI()).isEqualTo("/uri-logout/val1/val2");
 	}
 
+	/**
+	 * spring-restdocs uses postprocessors to do its trick. It will work only if these are merged together
+	 * with our request builders. (gh-7572)
+	 * @throws Exception
+	 */
+	@Test
+	public void postProcessorsAreMergedDuringMockMvcPerform() throws Exception {
+
+		RequestBuilder requestBuilder = logout();
+
+		MockHttpServletRequestBuilder defaultRequestBuilder = MockMvcRequestBuilders.get("/");
+		defaultRequestBuilder.with(new MockPostProcessor());
```

Comment:
Similar to the other test, this doesn't seem to be doing anything since if we remove the line above the test will still pass. I think we should likely use `MockMvc` here as I suggested above.